src/components/DateRangePicker.jsx

```diff
@@ -125,9 +136,44 @@ export default class DateRangePicker extends React.Component {
     const { onFocusChange } = this.props;
     if (!this.isOpened()) return;
 
+    this.setState({
+      isDateRangePickerInputFocused: false,
+      isDayPickerFocused: false,
+      showKeyboardShortcuts: false,
+    });
+
     onFocusChange(null);
   }
 
+  onDateRangePickerInputFocus(focusedInput) {
+    const { onFocusChange, withPortal, withFullScreenPortal } = this.props;
+
+    if (focusedInput) {
+      const moveFocusToDayPicker = withPortal || withFullScreenPortal;
```

Comment:
Okay, let's have this occur always for portals and also always on touch devices.